src/objecttypes/core/admin.py

```diff
@@ -89,6 +94,13 @@ class ObjectTypeAdmin(admin.ModelAdmin):
     search_fields = ("uuid",)
     inlines = [ObjectVersionInline]
 
+    change_list_template = "admin/core/objecttype/object_list.html"
+
+    def get_urls(self):
+        urls = super().get_urls()
+        my_urls = [path("import-github-url", self.import_github_url, name="import_github_url")]
```

Comment:
```suggestion
        my_urls = [path("import-github-url", self.admin_site.admin_view(self.import_github_url), name="import_github_url")]
```
Without this wrapper all unauthorized users could access this url